mailbox/plugin/deleted-messages-vault/pom.xml

```diff
@@ -85,10 +85,18 @@
             <artifactId>blob-memory</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-json</artifactId>
```

Comment:
why do you need this one if you import james-server-task-json? Do you use it directly?

Reply:
I fixed this in another PR and removed the associated commit from this one